### PR TITLE
Assignment colors

### PR DESCRIPTION
**Pull request by Fredasaurus**
Matches the colors for the entries on the calendar.

Shifts to using `data-assignment-type={planType}`.  Will need to continue refactoring in a separate pr.  Task plans seem to have a lot of repeated styling between types, for example.

screen shot 2015-08-14 at 11 26 50 am

screen shot 2015-08-14 at 11 26 58 am

screen shot 2015-09-11 at 7 52 26 pm
screen shot 2015-09-11 at 7 55 48 pm

### CODE REVIEW

**Comment by pandafulmanda** [comment]
Slick :smile: is this ready for review?  would it be too much or would it make sense to take the header colors into the homework builder top green title bar as well?

**Comment by Fredasaurus** [comment]
Yes.

It might work, or we could use the full strength version of the color. Let's try both.

**Comment by Fredasaurus** [comment]
Quick look modals and builders.

screen shot 2015-08-17 at 3 18 05 pm

screen shot 2015-08-17 at 3 18 18 pm

**Comment by pandafulmanda** [comment]
@Fredasaurus, is this ready for review?  Also, are we sure about this super bright background?

**Comment by Fredasaurus** [comment]
let's review. I'll swing by.

File: resources/styles/components/course-calendar/add.less [hunk 1]
```diff
@@ -7,4 +7,33 @@
...
+    &.active,
+    &.open {
+    	border-left: 5px solid @plan-color;
+      background: fade(@plan-color, 20%);
+      transition: background .1s ease-in, border .1s ease-in;
+    }
+  }
+
+  li {
+    a[href*="readings"] { .x-add-plan(@reading-color); }
```
Comment (philschatz) [inline resources/styles/components/course-calendar/add.less]: seems like we could provide a `[data-assignment-type='reading']` attribute on all the elements to make this easier/consistent

File: resources/styles/components/task-plan/index.less [hunk 2]
```diff
@@ -8,3 +8,9 @@
 @import './select-topics';
 @import './chapter-section';
 @import './footer';
+
+.task-plan {
+  &[data-assignment-type=reading] .dialog > .panel-heading { .tutor-plan(header, 'reading'); }
```
Comment (philschatz) [inline resources/styles/components/task-plan/index.less]: Would a http://lesscss.org/features/#detached-rulesets-feature remove the duplicated child selectors?

**Pull request merged**